app/src/main/java/com/keylesspalace/tusky/MainActivity.kt

```diff
@@ -291,21 +304,30 @@ class MainActivity : BottomSheetActivity(), ActionButtonActivity, MenuProvider {
 
         onBackPressedDispatcher.addCallback(this@MainActivity, onBackPressedCallback)
 
-        if (
-            Build.VERSION.SDK_INT >= 33 &&
-            ContextCompat.checkSelfPermission(this@MainActivity, Manifest.permission.POST_NOTIFICATIONS) != PackageManager.PERMISSION_GRANTED
-        ) {
-            ActivityCompat.requestPermissions(
-                this@MainActivity,
-                arrayOf(Manifest.permission.POST_NOTIFICATIONS),
-                1
-            )
-        }
-
         // "Post failed" dialog should display in this activity
         draftsAlert.observeInContext(this@MainActivity, true)
     }
 
+    override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<out String>, grantResults: IntArray) {
```

Comment:
Could use the newer `ActivityResultContracts` api instead. See `ViewMediaActivity` for an example.

Reply:
Done